tools/cgrep.py

```diff
@@ -1,93 +0,0 @@
-# Copyright 2011 Google Inc.
```

Comment:
Why the move? You should be able to set things up to import from the tools directory correct?